docs/src/man/comparisons.md

```diff
@@ -1,77 +1,191 @@
 # Comparisons
 
-This section compares DataFrames.jl with other data manipulation frameworks.
+This section compares DataFrames.jl with other data manipulation frameworks in Python, R, and Stata.
+
+A sample data set can be created using the following code:
+
+```julia
+df = DataFrame(id = 'a':'f', grp = repeat(1:2, 3), x = 6:-1:1, y = 4:9, z = [3:7; missing])
+df2 = DataFrame(grp = 1, w = 10)
+```
+
+## Comparison with the Python package pandas
+
+The following table compares the main functions of DataFrames.jl with the Python package pandas (version 1.1.0):
+
+```python
+>>> df = pandas.DataFrame({'grp': [1, 2, 1, 2, 1, 2],
+...                        'x': range(6, 0, -1),
+...                        'y': range(4, 10),
+...                        'z': [3, 4, 5, 6, 7, None]},
+...                       index = list('abcdef'))
+>>> df
+   grp  x  y    z
+a    1  6  4  3.0
+b    2  5  5  4.0
+c    1  4  6  5.0
+d    2  3  7  6.0
+e    1  2  8  7.0
+f    2  1  9  NaN
+```
+
+By comparison, this pandas data frame has `a` to `f` as row indices rather than a separate `id` column.
```

Comment:
```suggestion
By comparison, this pandas data frame has `a` to `f` as row indices rather than a separate `id` column.
In the DataFrames.jl examples we use `findfirst` function as row index in pandas
in general allows duplicates and in such case a row corresponding to the first
occurrence of a value in the row index is returned.
In DataFrames.jl in general a `groupby` function should be used to set a row index
to a `DataFrame`; opposed to pandas it then returns all rows matching an index in
case of duplicates.
```